samples/c2pa/C2paPlayer/C2paMenu/C2paMenu.js

```diff
@@ -48,23 +54,39 @@ export var C2PAMenu = function () {
         //Functions to access the c2pa menu items from the c2pa manifest
         c2paItem: function (itemName, c2paStatus, compromisedRegions = []) {
             const verificationStatus = c2paStatus.verified;
-            let manifest = null;
+            let manifest, producer, socialMedia = null;
             try {
                 manifest = c2paStatus.details.video.manifest;
             } catch (error) {
                 console.error('[C2PA] Manifest does not exist');
             }
-
             if (manifest != null && manifest['manifestStore'] != null) {
                 if (itemName == 'SIG_ISSUER') {
-                    return manifest['manifestStore']['activeManifest'][
-                        'signatureInfo'
-                    ]['issuer'];
+                    return manifest.manifestStore.activeManifest?.
```

Comment:
It might be less verbose to store `manifest.manifestStore.activeManifest` as a variable (such as `activeManifest`) that you can call below (such as `activeManifest?.signatureInfo?.time`)